### DISCUSSION TITLE
Mac OS 9.2 - "There is not enough space on the disc to complete the installation"

### DISCUSSION THREAD

**Discussion by steveparry1** [utmapp/UTM#6414] [General]
This is the error I get when trying to install MS Word 5 - an application from 1994ish. I'm told that the storage size is set way too high - about 10Gb. Can anyone suggest a suitable storage size to set in UTM settings? Thank you.

**Comment by lgrachov** [comment]
Somewhere about ~25 GB should work. You can also do half of the available space on your hard drive.
